core/src/main/java/ar/com/wolox/wolmo/core/fragment/WolmoFragmentHandler.java

```diff
@@ -57,12 +53,20 @@
     private boolean mVisible;
     private Unbinder mUnbinder;
 
-    @Inject T mPresenter;
-    @Inject ToastFactory mToastFactory;
-    @Inject Logger mLogger;
+    private @Nullable T mPresenter;
```

Comment:
I like this but we should add a `requirePresenter()` method that returns a `Presenter`, if it's not `null`, or throws an NPE if it doesn't. This method should return a `@NonNull` `Presenter` which is nice for both Java and Kotlin.

Reply:
Done